### PR TITLE
Graph profile drawer

### PR DESCRIPTION
**Pull request by wingfeatherwave**
Screenshot 2024-06-18 at 4 30 41 PM

### CODE REVIEW

File: src/pages/GiveParty/PartyProfileContent.tsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+            {
+              profile_id: targetProfile.id,
+            },
+            {
+              username: true,
+            },
+          ],
+        },
+        {
+          operationName: 'fc_profile',
```
Comment (crabsinger) [inline src/pages/GiveParty/PartyProfileContent.tsx:56]: more specific operation name so the logs indicate this is the file originating this request

File: src/pages/GiveParty/PartyProfileContent.tsx [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+          links: true,
+          description: true,
+          reputation_score: {
+            total_score: true,
+          },
+        },
+      ],
+    },
+    {
+      operationName: 'coLinks_profile',
```
Comment (crabsinger) [inline src/pages/GiveParty/PartyProfileContent.tsx:170]: same as above

**Comment by crabsinger** [comment]
@wingfeatherwave a few simple comments

**Review by crabsinger — DISMISSED** [review]

File: src/pages/GiveParty/PartyProfile.tsx [hunk 3]
```diff
@@ -1,21 +1,12 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
```
Comment (crabsinger) [inline src/pages/GiveParty/PartyProfile.tsx:1]: @wingfeatherwave any unsued vars still?

Comment (wingfeatherwave) [reply src/pages/GiveParty/PartyProfile.tsx]: there was one in the loading.  removed it

**Review by crabsinger — APPROVED** [review]

**Pull request merged**